_docs-sources/iac/usage/customizing-modules.md

```diff
@@ -0,0 +1,59 @@
+# Customizing Modules
+
+We strive to keep the Infrastructure as Code Library flexible and unopinionated, however there may be instances where you need to customize a module to your specific use case or a company policy that does not allow you to pull code from an external source.
```

Comment:
```suggestion
We strive to keep the Infrastructure as Code Library flexible and generic enough to serve common use cases, however there may be instances where you need to customize a module to your specific use case or a company policy that does not allow you to pull code from an external source.
```

Reply:
NIT: I think `unopinionated` is particularly problematic given that we advertise being opinionated as a key feature throughout the rest of our marketing materials. This could change rapidly, but feels like it would be jarring to folks paying attention.

Reply:
Good call! I lifted this from some existing docs but felt not-so-great about the usage of `unopinionated`.